src/options.cpp

```diff
@@ -188,6 +196,8 @@ options parse_options(int argc, char** argv) {
       o.version = true;
     } else if (parser.match_flag_option("--lsp-server"sv, "--lsp"sv)) {
       o.lsp_server = true;
+    } else if (parser.match_flag_option("--stdin"sv, "--s"sv)) {
```

Comment:
Nit: I don't like the `--s` option. I'd get rid of it, and only allow `--stdin`:
```suggestion
    } else if (parser.match_flag_option("--stdin"sv, "--stdin"sv)) {
```